std/haxe/io/Bytes.hx

```diff
@@ -73,6 +79,9 @@ class Bytes {
 		#end
 	}
 
+	/**
+	  Copies `len` bytes from `src` into this instance. 
```

Comment:
What about the other arguments? `len` bytes are copied, that's true. The destination is `pos` bytes into `this` instance, and the source is `srcpos` bytes into `src`.